auth-server/test/resolvers/mutation/addressLinkStart.spec.ts

```diff
@@ -59,6 +60,34 @@ describe('addressLinkStart mutation', () => {
 		expect(res.message).to.equal(messages.ADDRESS_LINKING_STARTED);
 	});
 
+	it('should not be able to start linking address if it already exists in db', async () => {
+		const network = NetworkEnum.KUSAMA;
+		const address = 'HNZata7iMYWmk5RvZRTiAsSDhV8366zq2YGb3tLH5Upf74F'; // Alice
+		const dbAddress = await Address
+			.query()
+			.allowInsert('[network, address, user_id, sign_message, verified]')
+			.insert({
+				address,
+				network,
+				sign_message: uuid(),
+				user_id: signupResult.user.id,
+				verified: false
```

Comment:
Because of this, the test should fail. It doesn't for another reason. I'd suggest to use another address rather than the one from Alice.

Reply:
fixed